### DISCUSSION TITLE
Domain taking too much time to propagate

### DISCUSSION THREAD

**Discussion by Yashhh999** [vercel/community#7086] [Help]
### Question

Hi , yesterday I applied a domain to my website and it is taking a lot of time propagate globally . It has taken now more than 12hrs .

**Comment by amyegan** [comment]
It can be normal to take 24-48 hours to see changes fully propagate through the network.

https://vercel.com/docs/projects/domains/troubleshooting#why-are-my-dns-records-taking-so-long-to-update
